util/spec_augment.py

```diff
@@ -0,0 +1,104 @@
+"""SpecAugment Implementation for Tensorflow.
+Related paper : https://arxiv.org/pdf/1904.08779.pdf
+In this paper, show summarized parameters by each open datasets in Tabel 1.
+-----------------------------------------
+Policy | W  | F  | m_F |  T  |  p  | m_T
+-----------------------------------------
+None   |  0 |  0 |  -  |  0  |  -  |  -
+-----------------------------------------
+LB     | 80 | 27 |  1  | 100 | 1.0 | 1
+-----------------------------------------
+LD     | 80 | 27 |  2  | 100 | 1.0 | 2
+-----------------------------------------
+SM     | 40 | 15 |  2  |  70 | 0.2 | 2
+-----------------------------------------
+SS     | 40 | 27 |  2  |  70 | 0.2 | 2
+-----------------------------------------
+LB : LibriSpeech basic
+LD : LibriSpeech double
+SM : Switchboard mild
+SS : Switchboard strong
+"""
```

Comment:
I think we should apply a single augmentation configuration to the entire combined train set, to avoid configuration explosion. I don't see anything in the SpecAugment paper that would indicate this is a bad idea.

Reply:
Is the idea that caching the train set would give you a single augmentation for all epochs, and not caching would compute a different augmentation for each epoch? (I think that would be fine but should be clear)